yarn.lock

```diff
@@ -11804,6 +11804,11 @@ passwd-user@^3.0.0:
   dependencies:
     execa "^1.0.0"
 
+path-browserify@^1.0.1:
```

Comment:
Mmmm, I love the smell of new dependencies in the morning....

:helicopter: :fire: